### PR TITLE
Soft constraint on risk

### PR DESCRIPTION
**Pull request by phschiele**
#27

### CODE REVIEW

File: cvx/markowitz/builder.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def build(self):
                 self.constraints[f"{name_model}_{name_constraint}"] = constraint
 
         problem = cp.Problem(self.objective, list(self.constraints.values()))
-        assert problem.is_dpp(), "Problem is not DPP"
+        # assert problem.is_dpp(), "Problem is not DPP"
```
Comment (phschiele) [inline cvx/markowitz/builder.py:148]: Currently rewriting to fix this before we merge.

**Comment by tschm** [comment]
Problem should raise an CvxError rather than an AssertionError

File: cvx/markowitz/builder.py (hunk 1 above)
Comment (tschm) [reply cvx/markowitz/builder.py:148]: I think we need this check back?

**Review by tschm — APPROVED** [review]
We have to be careful with the dpp check

**Comment by tschm** [comment]
@phschiele Usually the author of the pull request performs the merge...

File: cvx/markowitz/builder.py (hunk 1 above)
Comment (phschiele) [reply cvx/markowitz/builder.py]: Done in [17a3484](https://github.com/cvxgrp/cvxmarkowitz/pull/118/commits/17a3484ef9ae780f9dc67ab2acac1aec2aec06b2)

**Comment by phschiele** [comment]
Closes #27

**Pull request merged**